slave-dotnet20/Dockerfile.rhel7

```diff
@@ -0,0 +1,30 @@
+FROM openshift3/jenkins-slave-base-rhel7
```

Comment:
that's an artifact of how we build internally vs what gets published not being quite aligned.  we'll need this to be openshift for our internal CI flow to work, for now.